### PR TITLE
TGUI tg Controls Info Window

### PR DESCRIPTION
**Pull request by Garash2k**
## About the PR 
Converts the tg Controls Info popup into TGUI
image

Under the hood, adds a `content_window` parameter to `tgui_alert` which, if set, will lookup for a part of the same name in the `AlertContentWindows` folder. This should be useful for antag popups! When making new parts, besides the part itself, all that needs to be done is adding an `import` and a `switch-case` in `AlertContentWindows/index.ts`.

## Why's this needed? 
Having one of the first popup someone sees upon connecting look modern = good

[UI][Rework]

### CODE REVIEW

File: tgui/packages/tgui/interfaces/AlertContentWindows/tgControls.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * @copyright 2023
+ * @author garash2k
+ * @license ISC
+ */
+import { AlertContentWindow } from './types';
+import { Box, Image } from '../../components';
+
+export const tgControls: AlertContentWindow = {
+  width: 470,
+  height: 317,
```
Comment (ZeWaka) [inline tgui/packages/tgui/interfaces/AlertContentWindows/tgControls.tsx:12]:
```suggestion
  width: 470,
  height: 320,
```
cleaner

File: code/modules/tgui/tgui_alert.dm [hunk 2]
```diff
@@ -33,8 +35,10 @@
 	if(length(items) > 3)
 		log_tgui(user, "Error: TGUI Alert initiated with too many items. Use a list.", "TguiAlert")
 		return tgui_input_list(user, message, title, items, timeout, autofocus)
-	var/datum/tgui_modal/alert = new(user, message, title, items, timeout, autofocus)
+	var/datum/tgui_modal/alert = new(user, message, title, items, timeout, autofocus, content_window)
 	alert.ui_interact(user)
+	if (!do_wait)
+		return
```
Comment (ZeWaka) [inline code/modules/tgui/tgui_alert.dm:41]: cleanup?

File: code/client.dm [hunk 3]
```diff
@@ -402,7 +402,7 @@
 #ifndef IM_TESTING_SHIT_STOP_BARFING_CHANGELOGS_AT_ME
 				//preferences.randomizeLook()
 				preferences.ShowChoices(src.mob)
-				src.mob.Browse(grabResource("html/tgControls.html"),"window=tgcontrolsinfo;size=600x400;title=TG Controls Help")
+				tgui_alert(src, content_window = "tgControls", title = "Use /tg/ style interface?", do_wait = FALSE)
```
Comment (ZeWaka) [inline code/client.dm:405]: why is title not in the tgControls also

File: code/modules/tgui/tgui_alert.dm (hunk 2 above)
Comment (Garash2k) [reply code/modules/tgui/tgui_alert.dm:41]: Ref stops existing after closing the popup
image

**Comment by Garash2k** [comment]
Recent thought: if we think that piggy backing off `AlertModal.tsx`+`tgui_alert.dm` to make this sort of AlertContentWindow system is less ideal than having it be a seperate tgui export from a separate DM file, I could make that change. At first I just liked getting to re-use stuff like the ok buttons and the onKeyDown event, but I can see the upside of not messing with the original alert too.

**Pull request merged**